src/validation.cpp

```diff
@@ -4268,10 +4268,10 @@ void CChainState::ProcessProposalEvents(const CBlockIndex* pindex, CCustomCSView
 
         switch(prop.type) {
             case CPropType::CommunityFundProposal:
-                majorityThreshold = attributes->GetValue(cfpMajority, chainparams.GetConsensus().props.cfp.majorityThreshold) / 10000;
+                majorityThreshold = attributes->GetValue(cfpMajority, chainparams.GetConsensus().props.cfp.approvalThreshold) / 10000;
```

Comment:
Same as above, we can use helpers `GetQuorumFromAttributes` and `GetMajorityFromAttributes` in validation

Reply:
fixing in next commit with further improvements